index.html

```diff
@@ -0,0 +1,225 @@
+            <span class="list-item item1">Canopy</span>
+            <span class="list-item">back end Dev</span>
+            <span class="list-item">2015</span>
+           </div>
+        </div>
+        <div class="text-container">
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </div>
+        <div class="languages-list-container">
+          <ul class="lang-container">
+            <li class="langage-item"><span>JavaScript</span></li>
+            <li class="langage-item"><span>HTML</span></li>
+            <li class="langage-item"><span>CSS</span></li>
+          </ul>
+        </div>
+        <button type="button" class="see-btn">See Project</button>
+      </li>
+
+      <li class="card-item flex-center-culomn">
+        <img class="card-image" src="img/Snapshoot-Portfolio.png" alt="Portfolio Portfolio">
+        <div class="list-conatiner">
+          <h2>Tonic</h2>
+           <div class="list">
+            <span class="list-item item1">Canopy</span>
+            <span class="list-item">back end Dev</span>
+            <span class="list-item">2015</span>
+           </div>
+        </div>
+        <div class="text-container">
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </div>
+        <div class="languages-list-container">
+          <ul class="lang-container">
+            <li class="langage-item">JavaScript</li>
+            <li class="langage-item">HTML</li>
+            <li class="langage-item">CSS</li>
+          </ul>
+        </div>
+        <button type="button" class="see-btn">See Project</button>
+      </li>
+
+      <li class="card-item flex-center-culomn">
+        <img class="card-image" src="img/Snapshoot-Portfolio.png" alt="Portfolio Portfolio">
+        <div class="list-conatiner">
+          <h2>Tonic</h2>
+           <div class="list">
+            <span class="list-item item1">Canopy</span>
+            <span class="list-item">back end Dev</span>
+            <span class="list-item">2015</span>
+           </div>
+        </div>
+        <div class="text-container">
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </div>
+        <div class="languages-list-container">
+          <ul class="lang-container">
+            <li class="langage-item"><span>JavaScript</span></li>
+            <li class="langage-item"><span>HTML</span></li>
+            <li class="langage-item"><span>CSS</span></li>
+          </ul>
+        </div>
+        <button type="button" class="see-btn">See Project</button>
+      </li>
+
+    </ul>
+    </div>
+
+
+ 
+
+</section>
+
+<section >
+<div class="rgt-container">
+  <ul>
+    <li>
+      <div class="container lg">
+        <h4 class="txt-lg">Languages</h4>
+        <div class="container-arrow-right">
```

Comment:
- [ ] Good job by trying to implement the icons list of skills, but something went bad. you didn't implement them as on the Figma design.
### Your design:
![image](https://user-images.githubusercontent.com/71874425/159133484-08e25c90-f1a1-4a8b-b98f-a0e5daec9cfe.png)

### Figma Design:
![image](https://user-images.githubusercontent.com/71874425/159133371-e38f5a5f-a00b-46a0-b9aa-240f77d69d5b.png)
Kindly, check the Figma design and fix this issue.

Reply:
- [ ] As asked by the previous reviewer. Kindly make changes according, to implementing the figma design.
- Your design:
<img width="374" alt="Screenshot 2022-03-20 at 1 01 38 AM" src="https://user-images.githubusercontent.com/70269507/159139987-68a26019-a520-4d24-9178-03bb8e91a5c6.png">

- Figma design:
<img width="301" alt="Screenshot 2022-03-20 at 1 03 54 AM" src="https://user-images.githubusercontent.com/70269507/159139961-d1456bfb-309c-4f8a-a559-b89851e8fddb.png">